test/nut/commands/force/org/org.nut.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import * as querystring from 'querystring';
+import { expect } from '@salesforce/command/lib/test';
+import { TestSession } from '@salesforce/cli-plugins-testkit';
+import { execCmd } from '@salesforce/cli-plugins-testkit';
+import { Duration } from '@salesforce/kit';
+import { asDictionary, AnyJson, get, getString, isArray } from '@salesforce/ts-types';
+
+const concatString = (a: string, b: string): string => a + b;
+
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+describe('Org Command NUT', function async() {
+  this.timeout(Duration.minutes(10).milliseconds);
+  let session: TestSession;
+  let defaultUsername: string;
+  let aliasedUsername: string;
+  let defaultUserOrgId: string;
+  let aliasUserOrgId: string;
+  before(() => {
+    session = TestSession.create({
+      project: { name: 'forceOrgList' },
+      setupCommands: [
+        'sfdx force:org:create -f config/project-scratch-def.json --setdefaultusername --wait 10',
+        'sfdx force:org:create -f config/project-scratch-def.json --setalias anAlias --wait 10',
+      ],
+    });
+
+    if (isArray<AnyJson>(session.setup)) {
+      defaultUsername = getString(session.setup[0], 'result.username');
+      defaultUserOrgId = getString(session.setup[0], 'result.orgId');
+      aliasedUsername = getString(session.setup[1], 'result.username');
+      aliasUserOrgId = getString(session.setup[1], 'result.orgId');
+    }
+  });
+
+  after(async () => {
+    await session?.clean();
+  });
+
+  describe('List Orgs', () => {
+    it('should list all orgs', () => {
+      const result = execCmd('force:org:list --json');
```

Comment:
Totally works.  FYI: Mike added TS generics to execCmd to reduce the amount of casting that has to be done, which pairs nicely when commands export their return type.

example: https://github.com/salesforcecli/plugin-user/blob/40eb04a31d446cf480c524d5ab245621d5dec987/test/allCommands.nut.ts#L36